Volume/Extensions/Url+Extension.swift

```diff
@@ -8,6 +8,18 @@
 
 import Foundation
 
+enum ValidURLHost {
+    case article
+    
```

Comment:
Why do you use an enum with only one case?  Will there be more cases in the future?